lldb/test/Shell/Commands/command-image-lookup-color.test

```diff
@@ -0,0 +1,25 @@
+# RUN: %clang_host -g %S/Inputs/main.c -o %t
```

Comment:
Pick one of the two test files, I can't tell which one is supposed to be the actual one (perhaps you forgot to `git rm` the previous one?).

Reply:
Yes, I believe that was the case. Removed the other test.